php/src/GherkinDialect.php

```diff
@@ -21,13 +21,17 @@
  */
 final class GherkinDialect
 {
+    /** @var non-empty-list<non-empty-string> */
+    private readonly array $stepKeywords;
```

Comment:
Previously, the merged list was built on each access (which means for each line of the file being parsed when attempting to match step lines). I'm now computing the merged list once and storing it, which is consistent with the way the Java implementation works.